career-readiness.md

```diff
@@ -6,16 +6,16 @@ The LEARN program focuses on readying our students for a career in technology. T
 
 Deploying a project is an exciting, yet often challenging process. There are usually small errors that cause deployments to fail or cause bugs in production. One solution to this is to deploy early and update the deployment as the code evolves.
 
-Submit the link to your deployed portfolio site:
+ <!-- Submit the link to your deployed portfolio site: https://markayel-portfoliooo.onrender.com/ -->
```

Comment:
Please add some text to your portfolio (even just a heading with your name is fine for now).  Blank sites are considered incomplete.